.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
+  - "11"
```

Comment:
Let's test it in 12, which will be LTS. 11 is an intermediate version. Not worth testing contracts in that one.

Reply:
Done